import.py

```diff
@@ -191,23 +162,13 @@ def mark_imported(dataset: str):
 
 
 logger.info(f"{city} bounding box {bbox}")
-logger.info(f"{city} country {country}")
-
-country = country.lower()
-# some countries are known in nominatim, osmextracts and geofabrik by different names
-if country in COUNTRIES:
-    country = COUNTRIES[country]
 
 # OSM data needs to be imported first, will create the database
 if "osm" in datasets:
     logger.info(f"--- Importing OSM data for {city} ---")
-    import_path = os.path.join(os.path.dirname(__loader__.path), "scripts", "import_osm.sh")
-    for continent in CONTINENTS:
-        # Nominatim does not provide us with the continent. Will have to do some guessing
-        if not os.system(f"{import_path} {continent} {country} {slug} {osm_extracts_api_key}"):
-            # success!
-            mark_imported("osm")
-            break
+    osm_importer = OsmImporter({"slug": slug, "city": city, "bbox": ", ".join([str(coord) for coord in bbox]), "logger": logger})
```

Comment:
Here, we have to call the importer a bit different from the other importers below. Perhaps change the function signature so all importers use the same parameters?